generate-stackbrew-library.sh

```diff
@@ -4,7 +4,8 @@ set -e
 declare -A aliases
 aliases=(
 	[2.8]='2'
-	[3.0]='3 latest'
+	[3.0]=''
```

Comment:
I was not sure about the exact format of that file, so I simply emptied the
aliases. I will remove that line if that's the canonical way.
Am 06.05.2016 17:25 schrieb "Tianon Gravi" <notifications@github.com>:

> In generate-stackbrew-library.sh
> <https://github.com/docker-library/redis/pull/57#discussion_r62345190>:
>
> > @@ -4,7 +4,8 @@ set -e
> >  declare -A aliases
> >  aliases=(
> >  	[2.8]='2'
> > -	[3.0]='3 latest'
> > +	[3.0]=''
>
> I think we should just remove this line altogether (since we don't need
> any aliases for 3.0 anymore now that 3.2 is GA).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker-library/redis/pull/57/files/719878e99c7aed841052fe1948aefb8314d57858#r62345190>
>